cmd/fleetctl/user.go

```diff
@@ -172,6 +175,126 @@ func createUserCommand() *cli.Command {
 	}
 }
 
+func createBulkUsersCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "import",
+		Usage: "Create bulk users",
+		UsageText: `This command will create a set of users in Fleet by importing a CSV file. Expected columns are: Name,Email,Password,SSO,API Only,Global Role,Teams. Created Users by default get Observer Role.
+		Password could be left blank, Users should receive an invitation link.`,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     csvFlagName,
+				Usage:    "csv file with all the users (required)",
+				Required: true,
+			},
+			configFlag(),
+			contextFlag(),
+			yamlFlag(),
+			debugFlag(),
+		},
+		Action: func(c *cli.Context) error {
+			client, err := clientFromCLI(c)
+			if err != nil {
+				return err
+			}
+
+			csvFilePath := c.String(csvFlagName)
+
+			csvFile, err := os.Open(csvFilePath)
+			if err != nil {
+				return err
+			}
+			defer csvFile.Close()
+			csvLines, err := csv.NewReader(csvFile).ReadAll()
+
+			if err != nil {
+				return err
+			}
+			for _, record := range csvLines[1:] {
+				name := record[0]
+				email := record[1]
+				password := record[2]
+				sso := c.Bool(record[3])
```

Comment:
I wouldn't use `c` to parse the CSV fields. You can use `strconv.ParseBool` here and on `apiOnly` below.